sass/edx/_variables.scss

```diff
@@ -175,238 +157,207 @@ $grid-breakpoints : (
-
-
+//$mark-bg: #fcf8e3 !default;
+//
+//
 // Tables
 //
 // Customizes the `.table` component with basic values, each used across all table variations.
-
+//
 //$table-cell-padding:            .75rem !default;
-//$table-sm-cell-padding:         .3rem !default;
+//$table-cell-padding-sm:         .3rem !default;
 //
 //$table-bg:                      transparent !default;
+//$table-accent-bg:               rgba($black,.05) !default;
+//$table-hover-bg:                rgba($black,.075) !default;
+//$table-active-bg:               $table-hover-bg !default;
 //
-//$table-inverse-bg:              $gray-dark !default;
-//$table-inverse-color:           $body-bg !default;
-//
-//$table-bg-accent:               rgba($black,.05) !default;
-//$table-bg-hover:                rgba($black,.075) !default;
-//$table-bg-active:               $table-bg-hover !default;
+//$table-border-width:            $border-width !default;
+//$table-border-color:            $gray-200 !default;
 //
-//$table-head-bg:                 $gray-lighter !default;
-//$table-head-color:              $gray !default;
+//$table-head-bg:                 $gray-200 !default;
+//$table-head-color:              $gray-700 !default;
 //
-//$table-border-width:            $border-width !default;
-//$table-border-color:            $gray-lighter !default;
+//$table-dark-bg:              $gray-900 !default;
+//$table-dark-accent-bg:       rgba($white, .05) !default;
+//$table-dark-hover-bg:        rgba($white, .075) !default;
+//$table-dark-border-color:    lighten($gray-900, 7.5%) !default;
+//$table-dark-color:           $body-bg !default;
 
 
 // Buttons
 //
 // For each of Bootstrap's buttons, define text, background and border color.
 
-$btn-padding-x:                  1rem !default;
-$btn-padding-y:                  .5rem !default;
-$btn-line-height:                1.25 !default;
+$input-btn-padding-y:       .5rem !default;
+$input-btn-padding-x:       .75rem !default;
+$input-btn-line-height:     1.25 !default;
+
+$input-btn-padding-y-sm:    .25rem !default;
+$input-btn-padding-x-sm:    .5rem !default;
+$input-btn-line-height-sm:  1.5 !default;
+
+$input-btn-padding-y-lg:    .5rem !default;
+$input-btn-padding-x-lg:    1rem !default;
+$input-btn-line-height-lg:  1.5 !default;
+
 $btn-font-weight:                $font-weight-normal !default;
-$btn-box-shadow:                 none !default;
-$btn-focus-box-shadow:           none !default;
-$btn-active-box-shadow:          none !default;
+$btn-box-shadow:                 inset 0 1px 0 rgba($white,.15), 0 1px 1px rgba($black,.075) !default;
+$btn-focus-box-shadow:           0 0 0 3px rgba(theme-color("primary"), .25) !default;
+$btn-active-box-shadow:          inset 0 3px 5px rgba($black,.125) !default;
 
-$btn-primary-color:              $white !default;
-$btn-primary-bg:                 $brand-primary !default;
-$btn-primary-border:             $btn-primary-bg !default;
+$btn-link-disabled-color:        $gray-600 !default;
```

Comment:
It would be good for this to be a theme color. Maybe we need a new "disabled" theme color.

Reply:
Added